src/hotspot/share/gc/shenandoah/shenandoahFreeSet.hpp

```diff
@@ -629,6 +629,7 @@ using idx_t = ShenandoahSimpleBitMap::idx_t;
   void establish_old_collector_alloc_bias();
   size_t get_usable_free_words(size_t free_bytes) const;
 
+  void log_freeset_stats(ShenandoahFreeSetPartitionId partition_id, LogStream& ls);
```

Comment:
`log_freeset_stats` should probably be `private`.

Reply:
I thought it was private already? The `private` starts from [here](https://github.com/openjdk/jdk/blob/master/src/hotspot/share/gc/shenandoah/shenandoahFreeSet.hpp#L478). Or, if you expand this section a bit to line 636, another `public` starts after these declaration.